packages/styled-system/src/utils/transforms.js

```diff
@@ -1,35 +1,57 @@
 import get from './get';
 import toCSSVariable from './toCSSVariable';
 
-export const getter = (scale, value, options) => {
-  const { context, props } = { ...options };
-  const prefix = props?.theme?.config?.prefix; // defaults to 'tonic'
-  const useCSSVariables = props?.theme?.config?.useCSSVariables; // defaults to false
-  const cssVariableMap = props?.theme?.__cssVariableMap;
+// Check if a value is a simple CSS variable
+// e.g. var(--tonic-spacing-1)
+const isSimpleCSSVariable = (value) => {
+  const re = /^var\(\s*([a-zA-Z0-9\-_]+)\s*\)$/;
+  return re.test(String(value ?? '').trim());
+};
+
+// Negate the value, handling CSS variables and numeric values
+const toNegativeValue = (scale, absoluteValue, options) => {
+  const useCSSVariables = !!(options?.props?.theme?.config?.useCSSVariables); // defaults to false
+  const n = getter(scale, absoluteValue, options);
 
-  const result = get(scale, value, value);
+  // Handle CSS variables for negative values
+  if (useCSSVariables && isSimpleCSSVariable(n)) {
+    // https://stackoverflow.com/questions/49469344/using-negative-css-custom-properties
+    return `calc(0 - ${n})`;
+  }
 
-  if (result === undefined) {
-    return result;
+  // Handle numeric value
+  if (typeof n === 'number' && Number.isFinite(n)) {
+    return n * -1;
   }
 
-  if (useCSSVariables) {
-    const contextScale = context?.scale;
+  return `-${n}`;
+};
+
+export const getter = (scale, value, options) => {
+  const prefix = options?.props?.theme?.config?.prefix; // defaults to 'tonic'
+  const useCSSVariables = !!(options?.props?.theme?.config?.useCSSVariables); // defaults to false
+  const cssVariableMap = options?.props?.theme?.__cssVariableMap;
```

Comment:
```suggestion
  const { config, __cssVariableMap } =  options?.props?.theme
  const prefix = config?.prefix; // defaults to 'tonic'
  const useCSSVariables = !!(config?.useCSSVariables); // defaults to false
  const cssVariableMap = __cssVariableMap;
```

Reply:
revised in https://github.com/trendmicro-frontend/tonic-ui/pull/869/commits/788cc15cbe779a12e5c5557e9045f6c76eea5404